### PR TITLE
Kernel/aarch64: Build most of the kernel source files!

### PR DESCRIPTION
**Pull request by FireFox317**
With these commits, we now build the same source files for the x86(_64) and aarch64 kernel!

### CODE REVIEW

File: Kernel/Devices/Audio/AC97.cpp [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #include <Kernel/Arch/Delay.h>
 #include <Kernel/Devices/Audio/AC97.h>
 #include <Kernel/Devices/DeviceManagement.h>
+#include <Kernel/InterruptDisabler.h>
```
Comment (nico) [inline Kernel/Devices/Audio/AC97.cpp:11]: to be build -> to be built

(In commit msg)

File: Kernel/GlobalProcessExposed.cpp [hunk 2]
```diff
@@ -622,6 +624,7 @@ class ProcFSOverallProcesses final : public ProcFSGlobalInformation {
         return {};
     }
 };
+#if ARCH(I386) || ARCH(X86_64)
```
Comment (nico) [inline Kernel/GlobalProcessExposed.cpp:627]: How are we going to remember to add this when the time comes?

Comment (FireFox317) [reply Kernel/GlobalProcessExposed.cpp:627]: I guess when someone tries to run `SystemMonitor`, it'll try to access `/proc/cpuinfo`, which will then fail, and someone can implement this :)

Comment (FireFox317) [reply Kernel/GlobalProcessExposed.cpp]: Actually, I think it is better to print something to the console, and then return an error. I've implemented that :^)

**Comment by FireFox317** [comment]
Has been rebased on top of @supercomputer7 procfs/sysfs changes. Also includes 36d5fec6ca931150ba2edec7236e86906ecf0180 which cleans up a bunch of includes in SysFS/Subsystems/Kernel/Processes.cpp.

**Comment by gunnarbeutner** [comment]
bender_neat

**Pull request merged**